### ISSUE TITLE
build for windows and linux options

### ISSUE THREAD

**Issue by dawondyifraw** [reactide/reactide#203]
By  default build is for MAC OSX 
can we add options to build for Linux and windows I have modify package.json to specify package=linux settings .
Can we add a documentation for such scenarios ? 

Thanks

**Comment by PwrSrg** [comment]
Agreed.  The website states "Windows 8, 9, 10", but there is NO Windows Release???

**Comment by rgoshen** [comment]
Unfortunately, the existing directions do not work for Windows.

**Comment by MudulOzan** [comment]
Just check out #175 

Reactideception:
![image](https://user-images.githubusercontent.com/29442243/142298909-60b69470-c32a-4994-90d4-71c2d6f6d5bd.png)
